first_problem.rb

```diff
@@ -0,0 +1,23 @@
+class FirstProblem
+
+  def initialize(input_array)
+    @numbers_array = input_array
+  end
+
+  def compute()
+    if @numbers_array.empty?
```

Comment:
Do you need this check here? If you delete this, and rerun the tests, they should all pass.

Reply:
TDD tryouts ^^ 
I did unit test for empty array first , but yes now that i look over it again there is no problem to remove the check.